src/main/resources/public/js/vue.js

```diff
@@ -0,0 +1,8326 @@
+/*!
```

Comment:
I will remove vuejs, because it shouldn't be in this merge (Spring added), it was a test.
We can add webjar: [http://www.webjars.org/documentation#springboot](http://www.webjars.org/documentation#springboot)